### PR TITLE
Buy more things

### PR DESCRIPTION
**Pull request by kirknorthrop**
Speak to Kirk before merging
Needs: 

- [x] Blurb on ticket page
- [x] T&C update
- [x] GDPR/privacy policy update

### CODE REVIEW

File: orders/models.py [hunk 1]
```diff
@@ -177,25 +172,41 @@ def build_order_rows(self):
...
+                app_label="extras", model="childrenticket"
+            )
+
+            ticket = ExtraItem.objects.build(
+                content_type=children_ticket_content_type,
+                owner=self.purchaser,
+                details=self.unconfirmed_details,
+            )
+            row = self.order_rows.build_for_item(ticket)
+            rows.append(row)
```
Comment (inglesp) [inline orders/models.py]: Add an `else: assert False` here?

File: orders/templates/orders/emails/order-confirmation.txt [hunk 2]
```diff
@@ -1,6 +1,10 @@
 Hi {{ purchaser_name }},
 
+{% if num_tickets %}
 You have purchased {{ num_tickets }} ticket{{ num_tickets|pluralize }} for PyCon UK 2018.
+{% else %}
+You have purchased {{ num_items }} item{{ num_items|pluralize }} for PyCon UK 2018.
+{% endif %}
```
Comment (inglesp) [inline orders/templates/orders/emails/order-confirmation.txt]: It'd be good to list what the items are, if possible.

File: requirements.txt [hunk 3]
```diff
@@ -30,7 +30,7 @@ psycopg2-binary==2.7.4
 pycodestyle==2.3.1        # via flake8
 pyflakes==1.6.0           # via flake8
 pytz==2018.3              # via django
-pyyaml==4.1               # via tablib
+pyyaml==3.12              # via tablib
```
Comment (inglesp) [inline requirements.txt]: Oof, I've just read about PyYAMLgate.

File: orders/templates/orders/emails/order-confirmation.txt (hunk 2 above)
Comment (kirknorthrop) [reply orders/templates/orders/emails/order-confirmation.txt]: I'll have a look - we put it in the receipt, so I can't see why we can't put it here.

File: orders/models.py (hunk 1 above)
Comment (kirknorthrop) [reply orders/models.py]: You know how I feel about asserts in code 😄

**Comment by kirknorthrop** [comment]
Thanks for reviewing! 

There are a couple of queries which make it quite hard to do more than one ticket at a time, and as the relative number of sales of these items will be low, it did not seem to be worth the effort to fix it with all the other upheaval. Ditto with things like conference dinner - the number of people attending more than one should be minimal, and I can use check boxes for that case anyway.

There's also a much greater use of content type than I am truly happy with given its slightly cumbersome syntax, but it seemed the best solution until ironcage19 😉

**Comment by inglesp** [comment]
> There are a couple of queries which make it quite hard to do more than one ticket at a time, and as the relative number of sales of these items will be low, it did not seem to be worth the effort to fix it with all the other upheaval. Ditto with things like conference dinner - the number of people attending more than one should be minimal, and I can use check boxes for that case anyway.

OK.  Should think it'll be worth adding a note to the site saying that you can only buy one extra item at once otherwise you'll get people asking.  (We did last year, especially with accommodation which may not be an issue this time around!)

> There's also a much greater use of content type than I am truly happy with
Yeah, I noticed this, but I think it's Good Enough For Now (TM).

**Pull request merged**